src/Components/Resource/ResourceBoardView.tsx

```diff
@@ -34,7 +34,7 @@ export default function BoardView() {
   const [isLoading, setIsLoading] = useState(false);
   const [showFilters, setShowFilters] = useState(false);
 
-  badge;
+  // badge;
```

Comment:
What is badge used for? @bodhish @brainless

Reply:
That code was there and I had removed it, strange. Let me double check @peeyush14goyal.

Reply:
Also, this has been updated @peeyush14goyal.

Reply:
Its still in comments

Reply:
@peeyush14goyal can you please double check, I can not see it in code.